vm/mx.vm/mx_vm.py

```diff
@@ -105,16 +107,26 @@ def __init__(self, suite, name, deps, components, is_graalvm, exclLibs, platform
                  **kw_args):
         self.components = components
         base_dir = base_dir or '.'
+        _src_jdk = _get_jdk()
         _src_jdk_base, _jdk_dir = _get_jdk_dir()
         _src_jdk_base = _src_jdk_base if add_jdk_base else '.'
 
+        _src_jdk_has_jre = not (_src_jdk.version.parts[0] >= 11)
+
         if base_dir != '.':
             self.jdk_base = '/'.join([base_dir, _src_jdk_base]) if _src_jdk_base and _src_jdk_base != '.' else base_dir
         else:
             self.jdk_base = _src_jdk_base
 
+        if _src_jdk_has_jre:
+            self.jre_base = '/'.join(self.jdk_base, 'jre')
+        else:
+            self.jre_base = self.jdk_base
+
+
         path_substitutions = mx_subst.SubstitutionEngine(mx_subst.path_substitutions)
         path_substitutions.register_no_arg('jdk_base', lambda: self.jdk_base)
+        path_substitutions.register_no_arg('jre_base', lambda: self.jdk_base)
```

Comment:
i suppose this should be `self.jre_base`?

Reply:
yep, fixed.